### PR TITLE
[4.0] Replacing tags in Privacy: Extension Capabilities

### PR DESCRIPTION
**Pull request by Kostelano**
### Summary of Changes
Go to Privacy -> Extension Capabilities, note that the information is displayed as a list. However, there are one-item lists that don't look very good.

**BEFORE / AFTER**

Screenshot_1

Screenshot_2

### CODE REVIEW

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on f193970205a1afba2a0ae336f54c38577bb2c50c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on f193970205a1afba2a0ae336f54c38577bb2c50c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f193970205a1afba2a0ae336f54c38577bb2c50c

Ups, haven't noticed the second test.

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by Quy** [comment]
Please fix language string/markup as a side-effect of this PR.

File: administrator/components/com_privacy/tmpl/capabilities/default.php [hunk 1]
```diff
@@ -34,11 +34,11 @@
 						<?php echo Text::_('COM_PRIVACY_MSG_EXTENSION_NO_CAPABILITIES'); ?>
 					</div>
 				<?php else : ?>
-					<ul>
+					<div>
 						<?php foreach ($capabilities as $capability) : ?>
-							<li><?php echo $capability; ?></li>
+							<p><?php echo $capability; ?></p>
```
Comment (Quy) [inline administrator/components/com_privacy/tmpl/capabilities/default.php]: Removing p tag should fix the issue. I am not at a computer to test or use div.

**Comment by Kostelano** [comment]
@Quy, thanks.

I corrected the code + made changes to the language constants, because the text after the edit "stuck together". Everything looks good now.

Screenshot_1

@YatharthVyas @ChristineWk @sandramay0905 

If you have time, please retest.

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on 03b19aad28ac9cb6d58e69f0f960fefb1c0b4b34<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 03b19aad28ac9cb6d58e69f0f960fefb1c0b4b34<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by brianteeman** [comment]
Sorry this is not correct. They are lists and should remain as lists.
Changing a list item to a paragraph changes the semantic meaning

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 03b19aad28ac9cb6d58e69f0f960fefb1c0b4b34<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33499">issues.joomla.org/tracker/joomla-cms/33499</a>.</sub>

**Comment by Kostelano** [comment]
Hmm. There is no point in arguing. But I see no reason to use a list in a place where the list is 1 line long.

**Comment by brianteeman** [comment]
semantics. For a list with one item it might not make much of a difference in semantics. Changing a list of three items to 3 paragraphs changes the semantic meaning.

**Pull request closed without merging**